### PR TITLE
Log runner job conclusion

### PR DESCRIPTION
**Pull request by enescakir**
It will help us determine decreases in the success rate in real time using our monitoring tools.

I also added a data center to identify data center-specific issues.

### CODE REVIEW

**Review by furkansahin — APPROVED** [review]

File: model/github_runner.rb [hunk 1]
```diff
@@ -36,10 +36,13 @@ def display_state
   end
 
   def log_duration(message, duration)
-    values = {ubid: ubid, label: label, repository_name: repository_name, duration: duration}
+    values = {ubid: ubid, label: label, repository_name: repository_name, duration: duration, conclusion: workflow_job&.dig("conclusion")}
```
Comment (velioglu) [inline model/github_runner.rb:39]: Do we need conclusion info each point we log duration? We might have additional log_conclusion function and call it from the billing record. Also, logging conclusion from log_duration function is a bit unintuitive.

Comment (enescakir) [reply model/github_runner.rb:39]: Actually, the duration is the execution time for `runner_completed` log. So it's not just conclusion log.

Comment (enescakir) [reply model/github_runner.rb:39]: `workflow_job&.dig("conclusion")` is `nil` if the job not completed yet

Comment (enescakir) [reply model/github_runner.rb:39]: I can add a condition check if `{ conclusion: nil }` feels wrong such as

```ruby
if (conclusion = workflow_job&.dig("conclusion"))
  values[:conclusion] = conclusion
end
```

Comment (enescakir) [reply model/github_runner.rb:39]: @velioglu Any comment?

**Review by velioglu — APPROVED** [review]

**Pull request merged**